src/tcp/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod socks4;
+pub mod socks5;
```

Comment:
To keep compatible with old versions, please re-export the inner types to `tcp` here.

Reply:
Sure done